build-system/tasks/coverage-map/index.js

```diff
@@ -24,10 +24,11 @@ let puppeteer;
 let explore;
 
 const coverageJsonName = argv.json || 'coverage.json';
+const serverPort = argv.port || 8000;
 const testUrl =
-  argv.url || 'http://localhost:8000/examples/everything.amp.html';
+  argv.url || `http://localhost:${serverPort}/examples/everything.amp.html`;
 const outHtml = argv.html || 'coverage.html';
-const scrollDistance = 100;
+const inputJs = argv.file || 'v0.js';
```

Comment:
will this work with the `.mjs` extension for module builds?

Reply:
With the `.mjs` extension, I was able to open up `source-map-explorer`, but then I was not able to get code coverage (through `--coverage`) for the `.mjs` extension.